lib/dotcom/trip_plan/map.ex

```diff
@@ -150,4 +197,13 @@ defmodule Dotcom.TripPlan.Map do
   def z_index(%{current: idx, start: idx}), do: 100
   def z_index(%{current: idx, end: idx}), do: 100
   def z_index(%{}), do: 0
+
+  defp tailwind_color("#003DA5"), do: "blue-500"
+  defp tailwind_color("#80276C"), do: "purple-500"
+  defp tailwind_color("#00843D"), do: "green-500"
+  defp tailwind_color("#ED8B00"), do: "orange-500"
+  defp tailwind_color("#DA291C"), do: "red-500"
+  defp tailwind_color("#7C878E"), do: "silver-500"
```

Comment:
📓 For now, we have to use the Metro colors because it will only accept those. We can change Metro to pass in an arbitrary hex code.